scopes/react/react/react.env.ts

```diff
@@ -364,10 +364,10 @@ export class ReactEnv
   icon = 'https://static.bit.dev/extensions-icons/react.svg';
 
   /**
-   * returns a paths to a function which mounts a given component to DOM
+   * returns the path to the compositions template
    */
   getMounter() {
-    return require.resolve('./mount');
+    return require.resolve('@teambit/react.ui.compositions-app');
```

Comment:
this might not exist, you need to make sure it's a dependency of the react env

Reply:
it's the same as the docs-app, and it doesn't appear anywhere.
Where should I put it?